EnableNETAnalyzers feature not working

This file has bugs in these lines:

In these two locations, there is an extra space in `$(EnableNETAnalyzers )` before the closing parenthesis, causing MSBuild to not find that property:
https://github.com/dotnet/sdk/blob/4edb1ebd7a4169cc7b39aecd4a2f198c22dcd2cc/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets#L22
https://github.com/dotnet/sdk/blob/4edb1ebd7a4169cc7b39aecd4a2f198c22dcd2cc/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets#L26

So even if I set `EnableNETAnalyzers` to true in my csproj, it gets overriden to `false` because this line overrides it:
https://github.com/dotnet/sdk/blob/4edb1ebd7a4169cc7b39aecd4a2f198c22dcd2cc/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets#L26

Once that spaces get removed, the DLLs are not found because their paths are relative to the project, not to the SDK:

https://github.com/dotnet/sdk/blob/4edb1ebd7a4169cc7b39aecd4a2f198c22dcd2cc/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets#L37
https://github.com/dotnet/sdk/blob/4edb1ebd7a4169cc7b39aecd4a2f198c22dcd2cc/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets#L41
https://github.com/dotnet/sdk/blob/4edb1ebd7a4169cc7b39aecd4a2f198c22dcd2cc/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets#L44

Error:
```
CSC : error CS0006: Metadata file '..\analyzers\Microsoft.CodeAnalysis.CSharp.NetAnalyzers.dll' could not be found [D:\testing\ConsoleCore\ConsoleCore.csproj]
CSC : error CS0006: Metadata file '..\analyzers\Microsoft.CodeAnalysis.NetAnalyzers.dll' could not be found [D:\testing\ConsoleCore\ConsoleCore.csproj]
```

So the fix is to prefix the 3 DLL paths with `$(MSBuildThisFileDirectory)` like this:
```xml
Include="$(MSBuildThisFileDirectory)..\analyzers\Microsoft.CodeAnalysis.VisualBasic.NetAnalyzers.dll"
```

Once fixed, it can find instances of the new Roslyn analyzers.

This bug is present in master and in Preview8.

cc @jmarolf @mavasani

--- a/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets
+++ b/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets
@@ -11,24 +11,24 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers )' == '' And 
+    <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers)' == '' And 
                                    '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And
                                    $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '5.0'))">true</EnableNETAnalyzers>
-    <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers )' == ''">false</EnableNETAnalyzers>
+    <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers)' == ''">false</EnableNETAnalyzers>
   </PropertyGroup>
-  <Import Project="..\analyzers\build\Microsoft.CodeAnalysis.NetAnalyzers.targets"
+  <Import Project="$(MSBuildThisFileDirectory)..\analyzers\build\Microsoft.CodeAnalysis.NetAnalyzers.targets"
           Condition="$(EnableNETAnalyzers)" />
   <ItemGroup Condition="$(EnableNETAnalyzers)">
     <Analyzer
       Condition="'$(Language)' == 'VB'"
-      Include="..\analyzers\Microsoft.CodeAnalysis.VisualBasic.NetAnalyzers.dll"
+      Include="$(MSBuildThisFileDirectory)..\analyzers\Microsoft.CodeAnalysis.VisualBasic.NetAnalyzers.dll"
       IsImplicitlyDefined="true" />
     <Analyzer
       Condition="'$(Language)' == 'C#'"
-      Include="..\analyzers\Microsoft.CodeAnalysis.CSharp.NetAnalyzers.dll"
+      Include="$(MSBuildThisFileDirectory)..\analyzers\Microsoft.CodeAnalysis.CSharp.NetAnalyzers.dll"
       IsImplicitlyDefined="true" />
     <Analyzer
-      Include="..\analyzers\Microsoft.CodeAnalysis.NetAnalyzers.dll"
+      Include="$(MSBuildThisFileDirectory)..\analyzers\Microsoft.CodeAnalysis.NetAnalyzers.dll"
       IsImplicitlyDefined="true" />
   </ItemGroup>
 </Project>

